app/views/devise/passwords/edit.html.erb

```diff
@@ -18,8 +24,8 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Change my password" %>
+    <%= f.submit "Change my password", class: "btn btn-lg btn-primary btn-block" %>
```

Comment:
All of the application's headings? I do not know.

All of this content will be be moved over to translations which will make it easier to maintain consistency with capitalization.